src/Utils/operationsUtils.ts

```diff
@@ -0,0 +1,27 @@
+import { Expense, Income } from "../interfaces/interfaces";
+
+export const getLimitToWaste = (
+  incomes: Income[],
+  savingTarget: number
+): number => {
+  let incomesQuantity = 0;
+
+  incomes.forEach((income) => (incomesQuantity += income.quantity));
```

Comment:
we could use reduce for this one

Reply:
perfect, will change it to reduce method